### PR TITLE
Add archive warning.

### PR DESCRIPTION
**Pull request by cahrens**
@edx/teaching-and-learning: please review. I'm sorry that I failed to make this change in the previous PR.

FYI, I'm following https://open-edx-proposals.readthedocs.io/en/latest/oep-0014.html

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,18 +1,16 @@
-Part of `edX code`__.
-
-__ http://code.edx.org/
```
Comment (cahrens) [inline README.rst]: This website no longer exists.

File: README.rst [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
+[ARCHIVED] edx ORA (Open Response Assessor)
+===========================================
 
 The ORA will take a submission from an xqueue installation, pass it through machine learning grading, peer grading, and staff grading as appropriate, and return a result to LMS.  This is to be used with the edx-platform and xqueue.  It allows for the assessment of open response problems on the edx platform.
 
 DEPRECATED
 ------------------------
 
-The edX team has officially transitioned all support for the Open Response Assessor (ORA) project to its second iteration (ORA2). The original ORA project will remain open on github, but the edX team will no longer be supporting new features on the old version of the product.
+This repository has been archived and is no longer supported—use it at your own risk.
```
Comment (robrap) [inline README.rst]: Nit: supported—use => "supported. Use"

File: README.rst [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
+This repository may depend on out-of-date libraries with security issues, and security
+updates will not be provided. Pull requests against this repository will also not be merged.
 
-Note that this repository uses out-of-date versions of libraries that have known security issues. If this repository were to be used, the first step would be to update libraries in the various requirements files to pick up security updates.
+The edX team has officially transitioned all support for the Open Response Assessor (ORA) project to its second iteration (ORA2).
```
Comment (robrap) [inline README.rst]: Do we want to provide a link to ora2 repo?

**Review by robrap — APPROVED** [review]
Because this is archived, these changes aren't critical.  But feel free to update as well.  I don't need to re-review.

**Review by andy-armstrong — APPROVED** [review]
👍

File: README.rst (hunk 3 above)
Comment (cahrens) [inline README.rst]: It's pretty easy to find. If you search the edx organization for "ora" or "ora2", you will immediately find it.

**Pull request merged**